src/async-wrap.cc

```diff
@@ -468,6 +468,13 @@ void AsyncWrap::QueueDestroyId(const FunctionCallbackInfo<Value>& args) {
   PushBackDestroyId(Environment::GetCurrent(args), args[0]->NumberValue());
 }
 
+void AsyncWrap::AddWrapMethods(Environment* env,
```

Comment:
I felt `AddWrapMethods` doesn't convey the semantics clearly, so I tried to decostruct, and see what felt wrong.

>> `Add` -> `Bind`
>
> Why? I’d disagree.

`Add` has (for me) the connotation of creating something new, and since `env->SetProtoMethod` AFAICT only creates JS binding for an existing native method `Bind` sounds more appropriate. But we can also extend the existing metaphor and use `Set`.

>> `Wrap` -> `AsyncHooks`
>
> Not the same things.

Agreed, but AFAIK `getAsyncId` and `asyncReset` are new and only used in the context of the hooks so `AsyncHooks` felt more focused.
Maybe `AsyncIDs` is even better.

So to sum another option `SetAsyncIDsMethods`?